Libraries/Components/DrawerAndroid/DrawerLayoutAndroid.android.js

```diff
@@ -159,8 +159,11 @@ var DrawerLayoutAndroid = React.createClass({
 
   render: function() {
     var drawStatusBar = Platform.Version >= 21 && this.props.statusBarBackgroundColor;
+    var drawerSubviewBackgroundColor = (this.props.style || {}).backgroundColor || 'white';
```

Comment:
To clarify, it's not `styles.drawerBackgroundColor` but `styles.backgroundColor`. Since `AndroidDrawerLayout` is already taking in `style={[styles.base, this.props.style]}`, someone who is setting the background color for the drawer can do so via `style` property.

It doesn't break, but it shows some kind of weird elevation artifact.

![image](https://cloud.githubusercontent.com/assets/5796213/14463139/7717a802-00fc-11e6-8c2c-aedb16323198.png)

But why would you want to pass in a number for `backgroundColor`?